Sources/Templates/AutoRegistering.swift

```diff
@@ -86,8 +86,7 @@ private extension AutoRegistering {
             let joinedInitComponents = initComponents.joined(separator: ", ")
             lines.append("\(registrationName).register { \(registeringType.name)(\(joinedInitComponents)) }".indent(level: 2))
         } else {
-            let isShared = registeringType.staticVariables.contains { $0.name == "shared" }
-            let instance = isShared ? ".shared" : "()"
+            let instance = "()"
```

Comment:
I'd skip this var and just directly set it
`lines.append("\(registrationName).register { \(registeringType.name)() }".indent(level: 2))`